latch/services/register/register.py

```diff
@@ -359,13 +362,40 @@ def _upload_pkg_image(ctx: RegisterCtx) -> List[str]:
 
 
 def _register_serialized_pkg(ctx: RegisterCtx, serialize_dir: Path) -> dict:
+    headers = {"Authorization": f"Bearer {ctx.token}"}
 
-    files = {"version": ctx.version.encode("utf-8")}
+    serialize_files = {"version": ctx.version.encode("utf-8")}
     for dirname, dirnames, fnames in os.walk(serialize_dir):
         for filename in fnames + dirnames:
             file = Path(dirname).resolve().joinpath(filename)
-            files[file.name] = open(file, "rb")
+            serialize_files[file.name] = open(file, "rb")
+
+    response = requests.post(
+        ctx.latch_register_api_url,
+        headers=headers,
+        files=serialize_files,
+    )
+
+    commit_files = {".workflow_name": ctx.pkg_root.name.encode("utf-8")}
+
+    if not (ctx.remote is None):
+        commit_files[".remote_name"] = ctx.remote.encode("utf-8")
+
+    for dirname, dirnames, fnames in os.walk(ctx.pkg_root):
+        for filename in fnames:
```

Comment:
you should probably ignore the data folder here

in the future we will have a `.latchignore` but ignoring everything in the data folder is fine for larger files we dont want to send over multipart / commit (for now)